File: backend/ecommerce/src/main/java/com/example/ecommerce/domain/valueobject/CategoriaArticulo.java
package com.example.ecommerce.domain.valueobject;

/**
 * Las 4 categorías del catálogo definidas en el SyRS (mockup IU-01 y
 * Sección 1.1): Facial, Corporal, Capilar y Maquillaje. No es una entidad
 * gestionable con su propio CRUD porque RF-02 no lo pide.
 */
public enum CategoriaArticulo {
    FACIAL,
    CORPORAL,
    CAPILAR,
    MAQUILLAJE
}

File: frontend/src/app/core/models/categoria-articulo.ts
/** Espejo del enum CategoriaArticulo del backend (RF-02/RF-03). */
export type CategoriaArticulo = 'FACIAL' | 'CORPORAL' | 'CAPILAR' | 'MAQUILLAJE';

export const ETIQUETAS_CATEGORIA: Record<CategoriaArticulo, string> = {
  FACIAL: 'Facial',
  CORPORAL: 'Corporal',
  CAPILAR: 'Capilar',
  MAQUILLAJE: 'Maquillaje',
};

export const CATEGORIAS: CategoriaArticulo[] = ['FACIAL', 'CORPORAL', 'CAPILAR', 'MAQUILLAJE'];
